cmd/cubic-cli/cmd/transcribe.go

```diff
@@ -404,9 +426,51 @@ func processResults(outputWriter io.Writer, resultsChannel <-chan outputs) {
 		}
 	}
 
-	// Write formatted results to the outputWritter
-	for uttID, results := range finalResults {
-		fmt.Fprintf(os.Stdout, "Timeline for '%s':\n%s\n\n", uttID, timeline.Format(results))
+	// Sort the segmentIDs for consistancy between runs.
```

Comment:
This could perhaps be simplified a little bit; something like:

```go
// collect and sort utterance IDs from finalResults for deterministic order in output
uttIDs := make([]string, len(finalResults)
i := 0
for k := range finalResults {
    uttIDs[i] = k
    i++
}

sort.Strings(uttIDs)
```

and then:
```
case "utterance-json":
    for _, uttID := range uttIDs {
        for idx, result := range finalResults[uttID] {
            if str, err := json.Marshal(result); err != nil {
        }
    }
```